FoursquareTopTests/VenueViewModel+Builder.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  VenueViewModel+Builder.swift
+//  FoursquareTop
+//
+//  Created by Fran on 9/21/16.
+//  Copyright © 2016 Karumi. All rights reserved.
+
+
+import Foundation
+import MapKit
+
+@testable import FoursquareTop
+
+extension VenueViewModel {
```

Comment:
No objections about it, let give him a chance to express himself before sticking to whatever we prefer :P